File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F08.P1.java
class Main {
    static int overflowAtMax() {
        // BEGIN PROBE F08.P1
        int m = Integer.MAX_VALUE;
        int o = m + 1;
        return o;
        // END PROBE F08.P1
    }

    public static void main(String[] args) {
        System.out.println(overflowAtMax());
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/rust/F08.P1.rs
fn overflow_at_max() -> i32 {
// BEGIN PROBE F08.P1
let m = i32::MAX;
let o = m.checked_add(1).unwrap_or(0);
o
// END PROBE F08.P1
}

fn main() {
    println!("{}", overflow_at_max());
}
